actionview/lib/action_view/helpers/tags/radio_button.rb

```diff
@@ -17,7 +17,10 @@ def render
           options = @options.stringify_keys
           options["type"]     = "radio"
           options["value"]    = @tag_value
-          options["checked"] = "checked" if input_checked?(options)
+          if input_checked?(options)
+            options["checked"] = "checked"
```

Comment:
I'm not _totally_ sold on what to do for `<input type="radio">` and `<input type="checkbox">` elements. My original line of thinking was that `<input type="radio">` collections only have a single `[checked]` element, so if it's underlying attribute is invalid, that element is the culprit.

For `<input type="checkbox">`, the value might be invalid for various reasons: either _because_ of a checked value, _despite_ a checked value, or because a value _was not_ checked.

Reply:
Based on [this article on Tenon.io’s blog](https://blog.tenon.io/accessible-validation-of-checkbox-and-radiobutton-groups), it appears that `aria-invalid` on `<input type="radio">`/`<input type="checkbox">` elements doesn’t matter too much.